tests/integration_test/download_integration_test.py

```diff
@@ -0,0 +1,30 @@
+"""integration Tests for satip.download.py."""
```

Comment:
For pytest, I think the file name needs to be `test_{extra name}.py` for it to run? But I could be wrong, its certainly common pratice to do this. 

This is probably the same with the rest of the tests